package.json

```diff
@@ -15,6 +15,9 @@
     "storage"
   ],
   "main": "./lib/azure-storage.js",
+  "browserify": {
+    "./lib/common/signing/sharedkey.js": "./lib/common/signing/sharedkey.browser.js"
```

Comment:
Is this necessary?

Reply:
Yes. Needs to explictly define the replacement during browserify bundle.